src/training/data.py

```diff
@@ -460,11 +611,14 @@ def get_synthetic_dataset(args, preprocess_fn, is_train, epoch=0, tokenizer=None
 
     return DataInfo(dataloader, sampler)
 
+
 def get_dataset_fn(data_path, dataset_type):
     if dataset_type == "webdataset":
         return get_wds_dataset
```

Comment:
Need to pass the new collate_fn for webadaset as well? (around  `dataloader = wds.WebLoader(`)